src/bump-stdlibs.jl

```diff
@@ -141,8 +141,8 @@ function _bump_single_stdlib(stdlib::StdlibInfo, config::Config, state::State)
                             "Branch: $(stdlib.branch)",
                             "Old commit: $(stdlib_current_commit_in_upstream_short)",
                             "New commit: $(stdlib_latest_commit_short)",
-                            "Julia version: $(repr(julia_version))",
-                            "$(stdlib.name) version: $(repr(stdlib_version))$(version_match ? "" : " (Does not match)")",
+                            "Julia version: v$(julia_version)",
+                            "$(stdlib.name) version: v$(stdlib_version)$(version_match ? "" : " (Does not match)")",
```

Comment:
Just realized that the error messages would print with a v before them too. We can just drop the v
```suggestion
                            "Julia version: $(julia_version)",
                            "$(stdlib.name) version: $(stdlib_version)$(version_match ? "" : " (Does not match)")",
```